caduceator.yaml

```diff
@@ -175,8 +175,28 @@ vegetaConfig:
   # client timeout in seconds
   clientTimeout: "5s"
 
-  # destination of where to send wrpMessage
-  wrpMessageDest: "event:device-status/mac:112233445566/offline"
+  # messageDetails allows for the custom wrp messages to be sent
+  messageDetails:
+    # wrp is the custom wrp message
+    wrp:
+      source: "source"
+      destination: "destination"
+      metadata:
+        - "some-key": "some-value"
+        - "another key": "another value"
+      transactionUUID: "uuid-string"
+    # payload contains key value pairs that will be marshaled into a json and converted into a byte string
+    payload:
+      - "payloadKey": "payloadVal"
+    # bootTimeDuration is a duration used in creating the boot-time time-stamp. This duration
+    # is added to the current time.
+    bootTimeDuration: "-2h"
+    # bootTimeDuration is a duration used in creating the birthdate time-stamp. This duration
+    # is added to the current time.
+    birthdateDuration: "-1h"
```

Comment:
I think both `bootTimeOffset` and `birthdateOffset` explain a bit more what these durations are used for.